web/packages/core/.gitignore

```diff
@@ -0,0 +1,3 @@
+/pkg/
+/docs/
+/tsd/
```

Comment:
I'm not sure what `tsd` is. Maybe something related to TypeScript?
If it's not relevant anymore, we can remove it.

Reply:
I guess it's typescript definitions, but not sure how/if it was ever generated.